### PR TITLE
Add Custom Fields search

### PR DESCRIPTION
**Pull request by ricardogarfe**
Enable Custom Fields API connectivity.
* Get all custom fields grouped by entity
* Search by entity type associated

Using [Teamleader API documentation](http://apidocs.teamleader.eu/customfields.php)

### CODE REVIEW

File: CustomFields/CustomField.php [hunk 1]
```diff
@@ -0,0 +1,185 @@
+<?php
+
+namespace SumoCoders\Teamleader\CustomFields;
+
+/**
+ * CustomField class
+ *
+ * @author         Ricardo <php-teamleader@sumocoders.be>
```
Comment (carakas) [inline CustomFields/CustomField.php]: do you have a sumocoders email?

Comment (turanct) [inline CustomFields/CustomField.php]: `@author` is what git is for :-) just remove it?

File: CustomFields/CustomField.php [hunk 2]
```diff
@@ -0,0 +1,185 @@
+<?php
+
+namespace SumoCoders\Teamleader\CustomFields;
+
+/**
+ * CustomField class
+ *
+ * @author         Ricardo Garcia Fernandez <ricardogarfe@gmail.com>
```
Comment (ricardogarfe) [inline CustomFields/CustomField.php]: Updated author using my email

File: CustomFields/CustomField.php [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+ * @version        1.0.0
+ * @copyright      Copyright (c) SumoCoders. All rights reserved.
+ * @license        BSD License
```
Comment (carakas) [inline CustomFields/CustomField.php]: can you remove these 3 lines?

Comment (ricardogarfe) [inline CustomFields/CustomField.php]: Done 👍

**Pull request merged**